appium/tests/data/index.ts

```diff
@@ -69,10 +69,16 @@ export const CommonData = {
   },
   notIntegrityProtected: {
     subject: 'not integrity protected - should show a warning and not decrypt automatically',
-    message: '',
+    message: 'Message is missing integrity checks (MDC). The sender should update their outdated software and resend.',
     senderEmail: 'flowcrypt.compatibility@gmail.com',
-    encryptionBadgeText: 'not encrypted',
-    signatureBadgeText: 'not signed'
+  },
+  keyMismatchUnexpectedly: {
```

Comment:
It should be just `keyMismatch`, not `keyMismatchUnexpectedly`, as we're testing that modal isn't shown